pkg/core/blockchain.go

```diff
@@ -467,8 +468,10 @@ func (bc *Blockchain) storeBlock(block *Block) error {
 	}
 
 	atomic.StoreUint32(&bc.blockHeight, block.Index)
+	BlockHeight.Set(float64(bc.blockHeight))
 	for _, tx := range block.Transactions {
 		bc.memPool.Remove(tx.Hash())
+		MempoolCapacity.Set(float64(bc.memPool.capacity))
```

Comment:
Capacity is not changed here in any way, it's relatively static. We care about `Count()` mostly.

Reply:
And looks like `Remove()` already does update metrics.